gen/aws/templates/cloudformation.json

```diff
@@ -725,7 +725,7 @@
             "InstanceProtocol" : "TCP"
           }],
         "HealthCheck" : {
-          "Target" : "HTTP:9090/_haproxy_health_check",
+          "Target" : "TCP:9090",
```

Comment:
I don't think we want to change this... it's specifically set up this way for marathon-lb.